src/Main.elm

```diff
@@ -0,0 +1,137 @@
+module Main exposing (main)
+
+import Browser
+import DoubleSlider as DoubleSlider exposing (..)
+import Html exposing (Html, button, div, text)
+import Html.Events exposing (onClick)
+import RangeSlider as RangeSlider exposing (..)
+import SingleSlider exposing (..)
+
+
+main : Program Flags Model Msg
+main =
+    Browser.element { init = init, update = update, view = view, subscriptions = subscriptions }
+
+
+
+-- MODEL
+
+
+type alias Model =
+    { singleSlider : SingleSlider.SingleSlider Msg
+    , doubleSlider : DoubleSlider.DoubleSlider Msg
+    }
+
+
+type alias Flags =
+    {}
+
+
+init : Flags -> ( Model, Cmd Msg )
+init flags =
+    let
+        model =
+            { singleSlider =
+                SingleSlider.init
+                    { min = 0
+                    , max = 1000
+                    , value = 500
+                    , step = 50
+                    , onChange = handleSingleSliderChange
+                    , valueFormatter = RangeSlider.defaultValueFormatter
+                    , minFormatter = RangeSlider.defaultFormatter
+                    , maxFormatter = RangeSlider.defaultFormatter
+                    }
+            , doubleSlider =
+                DoubleSlider.init
+                    { min = 0
+                    , max = 1000
+                    , lowValue = 500
+                    , highValue = 750
+                    , step = 50
+                    , onLowChange = handleDoubleSliderLowChange
+                    , onHighChange = handleDoubleSliderHighChange
+                    , valueFormatter = RangeSlider.defaultValueFormatter
+                    , minFormatter = RangeSlider.defaultFormatter
+                    , maxFormatter = RangeSlider.defaultFormatter
+                    , currentRangeFormatter = DoubleSlider.defaultCurrentRangeFormatter
+                    , overlapThreshold = 1
```

Comment:
I think if we move the formatters out of the init functions then the PR would be good to go. :)

For example having some functions like `withMinFormatter` which changes the formatter being used which on init is the default one.

Maybe `overlapThreshold` as well... 🤔 Not really sure what `overlapThreshold` does, but the idea is that anything that is not necessary to make the component work or where we can provide a default then it should not be in the `init` function. This is so that we can add remove things from the config without changing the `init` function signature and also making the `init` function take less options.

Reply:
Yeah, makes sense!